### ISSUE TITLE
TypeError: input.includes is not a function

### ISSUE THREAD

**Issue by phillipstemann** [nuxt-modules/tailwindcss#505]
### Version
@nuxtjs/tailwindcss: 5.3.0
nuxt: 2.15.8

### My Tailwind config
`/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./components/**/*.{js,vue}",
    "./layouts/**/*.vue",
    "./pages/**/*.vue",
    "./plugins/**/*.js",
    "./nuxt.config.js",
  ],
  theme: {
    extend: {},
  },
  plugins: [
    require('@tailwindcss/forms')
  ],
}`

### Steps to reproduce
I've been using version 5.1.3 without any issues, but the moment I upgraded to 5.2.0 and now also 5.3.0 then, I get the error: TypeError: input.includes is not a function

Version 5.1.3 still works perfectly.

Here are the full error logs:
$ nuxt
ℹ Using default Tailwind CSS file from runtime/tailwind.css                                                                                                                    nuxt:tailwindcss

 FATAL  input.includes is not a function                                                                                                                                                       

  at normalizeWindowsPath (node_modules/pathe/dist/index.mjs:2:14)
  at normalize (node_modules/pathe/dist/index.mjs:17:10)
  at resolvePath (node_modules/@nuxt/kit/dist/index.mjs:340:10)
  at node_modules/@nuxtjs/tailwindcss/dist/module.mjs:166:75
  at Array.map (<anonymous>)
  at setup (node_modules/@nuxtjs/tailwindcss/dist/module.mjs:166:64)
  at async ModuleContainer.normalizedModule (node_modules/@nuxt/kit/dist/index.mjs:607:5)
  at async ModuleContainer.addModule (node_modules/@nuxt/core/dist/core.js:239:20)
  at async ModuleContainer.ready (node_modules/@nuxt/core/dist/core.js:51:7)
  at async Nuxt._init (node_modules/@nuxt/core/dist/core.js:478:5)

   ╭─────────────────────────────────────────────────╮
   │                                                 │
   │   ✖ Nuxt Fatal Error                            │
   │                                                 │
   │   TypeError: input.includes is not a function   │
   │                                                 │
   ╰─────────────────────────────────────────────────╯

error Command failed with exit code 1.

**Comment by pencilcheck** [comment]
very likely caused by this commit:

https://github.com/nuxt-modules/tailwindcss/compare/v5.2.0...v5.3.0

Where they upgraded nuxtjs/kit and it might broke the thing

**Comment by phillipstemann** [comment]
Yes, you're right.

**Comment by pencilcheck** [comment]
but for me it is weird since I tried to downgrade but still doesn't work from module point of view using this.addModule

**Comment by pencilcheck** [comment]
@phillipstemann btw you should also print out the line 
```
    if (typeof cssPath === "string") {
      if (existsSync(cssPath)) {
        logger.info(`Using Tailwind CSS from ~/${relative(nuxt.options.srcDir, cssPath)}`);
        resolvedCss = cssPath;
      } else {
        logger.info("Using default Tailwind CSS file from runtime/tailwind.css");
        resolvedCss = createResolver(import.meta.url).resolve("runtime/tailwind.css");
      }
    }
    nuxt.options.css = nuxt.options.css ?? [];
    console.log('nuxt.options.css', nuxt.options.css)
    const resolvedNuxtCss = await Promise.all(nuxt.options.css.map((p) => resolvePath(p)));
```
and see for yourself what is wrong, for me somehow nuxt-highlightjs is an object instead of a string in the css array, and an object is not normalizable for the path and that's why it errored out.

**Comment by mena234** [comment]
@pencilcheck  @phillipstemann 
This should be fixed in the new version 6.1.3.
[pull request](https://github.com/nuxt-modules/tailwindcss/pull/557#issue-1434681092)
